com.newbreedsoftware.circuslinux.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
This should go upstream.

Reply:
upstream does not accept changes anymore. last update 2001

Reply:
It's not a blocker but still, patches and extra stuff should always go upstream if possible.

Reply:
I understand but I have been building the fedora rpm for this since long time on OBS (open build service), debian, openSUSE all carry patches for this since 2001 (where upstream went probably AWOL - or moved to other tasks - I hope AWOL is user correctly here using it the first time... - `https://www.google.com/search?q=awol+meaning&oq=awol+meaning`)